lib/active_admin/resource_controller.rb

```diff
@@ -1,6 +1,7 @@
 require 'active_admin/resource_controller/action_builder'
 require 'active_admin/resource_controller/data_access'
 require 'active_admin/resource_controller/decorators'
+require 'active_admin/resource_controller/forms'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.